src/containerapp/azext_containerapp/custom.py

```diff
@@ -1480,7 +1481,25 @@ def set_workload_profile(cmd, resource_group_name, env_name, workload_profile_na
     return update_managed_environment(cmd, env_name, resource_group_name, workload_profile_type=workload_profile_type, workload_profile_name=workload_profile_name, min_nodes=min_nodes, max_nodes=max_nodes)
 
 
-def patch_list(cmd, resource_group_name=None, managed_env=None, show_all=False):
+def patch_list(cmd, resource_group_name=None, managed_env=None, container_app_name=None, show_all=False):
+    if container_app_name:
+        # Use cloud patching if container app name is provided
+        logger.warning("Container app name is provided. List cloud patches available for the container app.")
+        logger.warning("Cloud patching is only supported in North Central US (Stage) now.")
+        app = show_containerapp(cmd, container_app_name, resource_group_name)
+        if app is None:
+            logger.error("Container app {0} not found in resource group {1}.".format(container_app_name, resource_group_name))
+            return
+        if app["location"] != "North Central US (Stage)":
+            logger.warning("Cloud patching is not supported in the location of the container app. Defaulted back to use local patching.")
```

Comment:
The cloud patching only supported in north central us(stage)? Why?

Reply:
As of now, the cloud patch is only enabled in ncus stage as the compute and scanning are only enabled there. We'll open up to more regions as we go, but we'll firstly enable this feature for ncus stage only.